src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
-public class MainApp {
+import javax.crypto.Mac;
 
+public class MainApp {
+    public void doWork () {
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
```

Comment:
You should create classes Truck, Bulldozer, Excavator that will inherit from Machine as task description required and override doWork and stopWork method.